dll/cpl/timedate/lang/bg-BG.rc

```diff
@@ -33,7 +33,7 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Сверяване от интернет"
 FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
-    COMBOBOX IDC_SERVERLIST, 65, 22, 117, 136, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    COMBOBOX IDC_SERVERLIST, 65, 22, 117, 136, CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
```

Comment:
Why, do we want users to enter any arbitrary string?

Reply:
Yes, selecting a custom NTP server makes sense.

Reply:
Entering user data is allowed by Windows Server 2003 and XP. This way the user can select some specific NTP server that is not already contained in the registry.

Reply:
ok.